CMakeOpenCLPythonPackageCache.txt

```diff
@@ -0,0 +1,13 @@
+
+set(__OPENCL_PYTHON_PACKAGE ON CACHE BOOL "" FORCE)
+
+set(USE_MPI OFF CACHE BOOL "")
+set(USE_OPENMP ON CACHE BOOL "")
+set(USE_GPU OFF CACHE BOOL "")
+set(USE_SWIG OFF CACHE BOOL "")
+set(USE_HDFS OFF CACHE BOOL "")
+set(USE_R35 OFF CACHE BOOL "")
+set(USE_TIMETAG OFF CACHE BOOL "")
+set(USE_DEBUG OFF CACHE BOOL "")
+set(BUILD_FOR_R OFF CACHE BOOL "")
+
```

Comment:
I don't have other comments to offer yet, but can you please be sure that `CMake` cache files are not checked into the repo?

Maybe add a `*Cache.txt` entry to `.gitignore`, I don't think that would conflict with anything we have intentionally checked in.

Reply:
Thanks for the early feedback.  

The file was meant as a cmake preload file, but since it's likely to cause further confusion, I've moved its contents into the main CMakeLists.txt file and removed the offending file.